src/main/java/model/player/Player.java

```diff
@@ -0,0 +1,75 @@
+package model.player;
+
+import java.util.List;
+import java.util.Objects;
+import model.card.Card;
+import model.card.Cards;
+
+public class Player {
+
+    private static final String INVALID_NAME_LENGTH = "플레이어 이름은 빈 값일 수 없습니다.";
+    private static final int ADD_CARD_CONDITION = 22;
+
+    private final String name;
+    private final Cards cards;
+
+    public Player(String name) {
+        this(name, new Cards(List.of()));
+    }
+
+    public Player(String name, Cards cards) {
+        validateEmptyName(name);
+        this.name = name;
+        this.cards = cards;
+    }
+
+    private void validateEmptyName(String name) {
+        if (name.isEmpty()) {
+            throw new IllegalArgumentException(INVALID_NAME_LENGTH);
+        }
+    }
+
+    public boolean isPossibleAddCard() {
+        int totalNumbers = cards.calculateTotalNumbers();
+        return totalNumbers < ADD_CARD_CONDITION;
+    }
+
+    public Player addCard(Card card) {
+        Cards addedCards = cards.add(card);
+        return new Player(name, addedCards);
+    }
+
+    public Player addCards(List<Card> cardsElement) {
```

Comment:
```suggestion
    public Player addCards(List<Card> cards) {
```
cards라고 스셔도 무방할것 같네요
멤버 변수와 혼동이 된다면 this 키워드를 이용하는게 일반적입니다

Reply:
원시값포장은 멤버변수에 해당되는 개념입니다.
파라미터는 고려안하셔도 됩니다!
원시값은 의미있는 동작을 수행하는 값이라면 해주면 좋을것 같아요
저라면 name과 같이 validation만 있다면 포장을 하지는 않을것 같아요
자동차게임에서 car의 postion과 같이 의미있는 동작과 검증이 필요한 값이라면 원시값 포장이 큰 의미가 있을것 같습니다!